graph_service_client.go

```diff
@@ -70,3 +72,8 @@ func (m *GraphBaseServiceClient) Me() *if6ffd1464db2d9c22e351b03e4c00ebd24a5353c
 	urlTplParams["user%2Did"] = "me-token-to-replace"
 	return if6ffd1464db2d9c22e351b03e4c00ebd24a5353cd70ffb7f56cfad1c3ceec329.NewUserItemRequestBuilderInternal(urlTplParams, m.requestAdapter)
 }
+
+// NewPageIterator returns an instance of PageIterator to process results
+func (m *GraphBaseServiceClient) NewPageIterator(pagedResult interface{}) (*coreutils.PageIterator, error) {
```

Comment:
holding on this comment https://github.com/microsoftgraph/msgraph-sdk-go/issues/257#issuecomment-1460215939